scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -35,7 +35,7 @@ def _get_run_numbers_for_key(current_mode_run_numbers, key):
 def _get_current_mode_dictionary(run_number_string, inst_settings):
     mapping_dict = get_cal_mapping_dict(run_number_string, inst_settings.cal_mapping_path)
     if inst_settings.mode is None:
-        ws = mantid.Load('POLARIS'+run_number_string+'.nxs')
+        ws = mantid.Load('POLARIS'+run_number_string)
```

Comment:
What's the reason for removing the .nxs prefix here?

Reply:
To allow loading from the archive "POLARIS98533.nxs" does not exist but searching "POLARIS98533" will find "POLARIS0098533.nxs" on the archive. We should check that this doesn't stop loading locally on Windows.